slides/control_structures.tex

```diff
@@ -0,0 +1,183 @@
+% Choose one to switch betweeen slides and handout
+\documentclass[]{beamer}
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Monetary Control Structures}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Monetary Control Structures}
+	Monetary units display three different control structures:
+	\vspace{1.5em}
+	\begin{figure}
+		\input{../assets/figures/structure.tex}
+	\end{figure}
```

Comment:
Maybe include also a the different opposite pairs per dimension briefly, i.e., competitive vs. monopolistic,  physical vs. virtual, etc.

Reply:
Maybe be the used images did not work as intended (or they need Fabian's explanations), but I meant to illustrate both dimensions for each pair: monopolistic (money printer) vs. competitive (miner); physical (currency), virtual (money on smartphone); transaction processing (intermediary vs peer-to-peer).

I hope this is clearer in the video.